docs/using/installing/index.rst

```diff
@@ -34,14 +34,20 @@ On Ubuntu, the Flocker CLI can be installed from the ClusterHQ repository:
 Other Linux Distributions
 -------------------------
 
+.. warning::
+
+   These are guidelines for installing Flocker on a Linux distribution which we do not provide native packages for.
+   These guidelines may require some tweaks, depending on the details of the Linux distribution in use.
+
 Before you install ``flocker-cli`` you will need a compiler, Python 2.7, and the ``virtualenv`` Python utility installed.
-On Fedora 20 you can install these by running:
+
+To install these with the ``yum`` package manager, run:
```

Comment:
To make this work in general with yum systems, replace @buildsys-build with gcc. If a particular OS requires additional dependencies, let the user figure them out.